packages/stark-ui/src/modules/table/components/_table.component.scss

```diff
@@ -19,6 +19,13 @@
       display: flex;
       justify-content: flex-end;
 
+      .button-global-filter {
+        opacity: 0.2;
```

Comment:
Why did you set `opacity: 0.2` and not `0.3` as is this the case for the columns ?
```css
.button-filter {
  opacity: 0.3;
}
```

Reply:
Yeah, I know, but as I explained to you, the opacity should be lighter because the base color in the header is darker than in the columns 😉